README.md

```diff
@@ -17,8 +17,38 @@ violet-gulls
 
 ## Installation and Setup
 
-_Fill this out_
+To install dependencies, type the following command in your terminal:
+``` npm install ```
 
+If you have PostgreSQL, create database by using command:
+``` npm run db:create ```
+
+Run this command to create your table migrations:
+``` npm run db:migrate ```
+
+Seed your database with test data:
+``` npm run db:seed ```
+
+To drop your database use command:
+``` npm run db:drop ```
+
+To reset your database by dropping, creating and seeding your database, use:
+``` npm run db:reset ```
+
+To load the REPL use:
+``` npm run repl ```
+
+To run quieres from the console, use:
+``` npm run console ```
 ## Usage and Examples
 
```

Comment:
I like the way you gave the instructions. The first should have been to clone the repo.